app/components/pages/onboarding-steps/Multistream.tsx

```diff
@@ -0,0 +1,127 @@
+import { Component } from 'vue-property-decorator';
+import TsxComponent, { createProps } from 'components/tsx-component';
+import electron from 'electron';
+import { OnboardingStep } from 'streamlabs-beaker';
+import { Inject } from 'services';
+import { UserService } from 'services/user';
+import { NavigationService } from 'services/navigation';
+import { $t } from 'services/i18n';
+import { RestreamService } from 'services/restream';
+import { StreamSettingsService } from 'services/settings/streaming';
+import { ToggleInput } from 'components/shared/inputs/inputs';
+import styles from './Multistream.m.less';
+
+class MultistreamProps {
+  continue: () => void = null;
```

Comment:
Is `null` the correct default prop here or should it be `() => {}`?

Reply:
not sure it makes a huge difference since it's a required prop but i can make that change

Reply:
Ah ok.  Why is a default value needed at all if it's a required prop?

Reply:
the way `createProps` works we need any prop to have non-undefined values